ds_judgements_public_ui/templates/includes/style_guide/line_heights.html

```diff
@@ -0,0 +1,12 @@
+<h3 id="line-heights">Line heights</h3>
+<p>These are the variables available to style your components:</p>
+
+<div class="style-guide__component-wrapper" >
+    <div class="style-guide__component-wrapper xl3-line-height">$typography-3xl-line-height</div>
```

Comment:
This feels like it's not a part of this PR.

Reply:
I think because the change is big github thinks it's a new file - it's just a rename of the file and class